Documentation/network/servicemesh/ingress-to-gateway/ingress-to-gateway.rst

```diff
@@ -0,0 +1,109 @@
+
+- Limited protocol support:
+  
+The Ingress API only supports HTTP and HTTPS traffic, and does not natively support other protocols such as TCP or UDP.
+The specification of Ingress API was simply too limited and not extensible enough. To address these technical limitations, 
+software vendors and developers created vendor-specific annotations to provide these specific capabilities. But using annotations 
+ended up creating inconsistencies from one Ingress Controller to another. This means that, if you wanted to start using a different 
+Ingress Controller over another, you would likely face some conversion issues as annotations tend to be vendor-specific.
+
+- Operational constraints:
+  
+Finally, the Ingress API suffers from operational constraints: it simply is not well-suited for multi-team clusters with shared load-balancing infrastructure.
+
+Benefits of Gateway API
+#######################
+
+The Gateway API was designed from the ground up to address the Ingress API limitations. The team behind the Gateway API is a Kubernetes SIG-Network project.
+
+.. note::
+
+    You can find out more information about the project on its `website <https://gateway-api.sigs.k8s.io/>`_.
+
+On the technical front, the Gateway API provides a centralized mechanism for managing and enforcing policies for external traffic, 
+including HTTP routing, TLS termination, traffic splitting/weighting, and header modification.
+
+Native support for these features means annotations are no longer needed to provide support for features that are commonly required
+for ingress traffic patterns. This means that Gateway API resources are more portable from one Gateway API implementation to another.
+
+When customization is required, Gateway API provides several flexible models, including specific extension points to enable diverse 
+traffic patterns. As extensions are added, the Gateway API team will keep looking for common denominators and will promote features 
+to the API conformance to avoid going back to some of the chaos seen with extending Ingress API resources.
+
+Finally, the Gateway API implementation was designed with role-based persona in mind. The Ingress model was based on a model where 
+it was assumed that developers managed and created Ingress and Services resources themselves.
+
+In many of the more complex deployments however, there are various personas involved:
+
+- Infrastructure Provider: responsible for the administration of the underlying infrastructure. For example, the cloud provider (AWS, Azure, GCP) when using 
+  their managed services or the infrastructure/network team when running Kubernetes on-premises.
+- Cluster Operator: responsible for the the administration of a cluster.
+- Application Developer: responsible for defining application configuration and service composition.
+
+By decomposing the Ingress API into several Gateway API objects, personas will be assigned the right access and privileges that their responsibilities require.
```

Comment:
Use present tense, use active voice

It's not clear to me what action and results you're describing here, especially the relationships between Ingress API functions (presumably?), Gateway API objects, and role-based personas. Reword this sentence to be more clear.
```suggestion
By deconstructing the Ingress API into several Gateway API objects, personas gain the access and privileges that their responsibilities require.
```